examples/svelte/sveltekit/src/routes/[slug].svelte

```diff
@@ -0,0 +1,90 @@
+<script context="module" lang="ts">
+	export const prerender = true;
+</script>
+
+<script lang="ts">
+	import { page } from '$app/stores';
+
+	import * as BuilderSDK from '@builder.io/sdk-svelte';
+
+	// TODO: enter your public API key
+	const BUILDER_PUBLIC_API_KEY = 'f1a790f8c3204b3b8c5c1795aeac4660'; // ggignore
+
+	// onMounted() {
```

Comment:
Should we remove or add more context to this commented stuff? It might cause confusion for people using this example

Reply:
I actually just removed it in a commit. It's redundant in Svelte. Good catch!